package-lock.json

```diff
@@ -0,0 +1,17384 @@
+{
```

Comment:
The project enjoys yarn.lock. so maybe we need to remove the file here and ensure yarn.lock track the package update well.